homework_3/homework3.py

```diff
@@ -0,0 +1,26 @@
+#1
+i = " I Love Python"
+print(i*42)
+#2
+age_in_month = 12*17
+print("month =", age_in_month) 
+#3
+age_in_years = 17
+#4
+my_age = "Му name is Dmytro and I’m 17 years old"
+#5
+comparison = 1
+z = (comparison ==1)
+x = (comparison >34)
+l = (comparison <=23)
+v = (comparison !=17)
+
+print(z,x,l,v)
+
+#6
+a=2
+b=5
+c=6
+
+d=(2*5+6)**2
```

Comment:
The final value should be a string. To create a string from integers you can transform integers to strings using str() function and concatenate them using `+` sign